build.gradle.kts

```diff
@@ -231,6 +231,11 @@ subprojects {
 						"https://twitch4j.github.io/javadoc"
 				)
 				locale = "en"
+
+				// additional javadoc tags
+				addStringOption("tag", "apiNote:a:API Note:")
+				addStringOption("tag", "implSpec:a:Implementation Requirements:")
+				addStringOption("tag", "implNote:a:Implementation Note:")
```

Comment:
it seems like `addStringOption` overwrites the previous value(s); javadoc.options only has `-tag 'implNote:a:Implementation Note:'`

we could do:
```kt
tags = listOf("apiNote:a:API Note:", "implSpec:a:Implementation Requirements:", "implNote:a:Implementation Note:")
```

or

```kt
addMultilineStringsOption("tag").value = listOf("apiNote:a:API Note:", "implSpec:a:Implementation Requirements:", "implNote:a:Implementation Note:")
```